src/data/graphql.tsx

```diff
@@ -8294,6 +8243,8 @@ export type SignQuotesMutationFn = ApolloReactCommon.MutationFunction<
  * const [signQuotesMutation, { data, loading, error }] = useSignQuotesMutation({
  *   variables: {
  *      quoteIds: // value for 'quoteIds'
+ *      successUrl: // value for 'successUrl'
+ *      failUrl: // value for 'failUrl'
  *   },
  * });
  */
```

Comment:
do we need the comments?

Reply:
No, but this whole file is generated so I can’t help it haha 😬